cypress/support/api_commands/channels.js

```diff
@@ -0,0 +1,148 @@
+function reqChannelBySlot(slot) {
+  return cy.request({
+    method: 'GET',
+    url: `/channels/slot:${slot}`,
+    headers: {
+      accept: 'application/json',
+    }
+  }).its('body');
+}
+Cypress.Commands.add('reqChannelBySlot', reqChannelBySlot);
+
+function reqSetChannelOutput(slot, value) {
+  return cy.request({
+    method: 'POST',
+    url: '/channels/slot:' + slot + '/output',
+    body: { value },
+    headers: {
+      'content-type': 'application/json',
+      accept: 'application/json',
+    }
+  });
+}
+Cypress.Commands.add('reqSetChannelOutput', reqSetChannelOutput);
+
+function reqSetChannelLiveID(slot, value) {
+  return cy.request({
+    method: 'POST',
+    url: '/channels/slot:' + slot + '/liveId',
```

Comment:
we should use string interpolation literals 
```suggestion
    url: `/channels/slot:${slot}/liveId`,
```